.github/CODEOWNERS

```diff
@@ -144,8 +140,6 @@ azure-pipelines.yml @coq/ci-maintainers
 /plugins/derive/        @aspiwack
 # Secondary maintainer @ppedrot
 
-/plugins/extraction/    @maximedenes
```

Comment:
This is problematic, we should find another one

Reply:
I think the absence of a maintainer for extraction should be discussed in a WG, if you want to get yourself our of this you have surely the right.

Reply:
TBH, the reason why I remove my name from here is that I had another look at extraction recently, and I don't understand the implementation. So I'm not competent as a maintainer. IIRC, I originally put my name as a secondary maintainer, hoping to learn more about the implementation.

Reply:
The only other person that changed this plugin recently beyond @letouzey is @JasonGross. But maybe @mattam82 or @herbelin have knowledge of extraction code as well. In any case, extraction seems like a critical component, and this is a reminder that we are currently without any maintainer for it.